app/controllers/bookmarks_controller.rb

```diff
@@ -61,7 +61,11 @@ def search
   def index
     if @bookmarkable
       access_denied unless is_admin? || @bookmarkable.visible?
-      @bookmarks = @bookmarkable.bookmarks.is_public.paginate(page: params[:page], per_page: ArchiveConfig.ITEMS_PER_PAGE)
+      @bookmarks = @bookmarkable.bookmarks.is_public
+      if is_admin?
+        @bookmarks += @bookmarkable.bookmarks.where(hidden_by_admin: true)
```

Comment:
I think this is a bit weird how I'm doing it, but I didn't see another way to include bookmarks that have been hidden by admin *and* the public bookmarks.